internal/store/pod.go

```diff
@@ -1318,6 +1320,60 @@ func createPodStatusPhaseFamilyGenerator() generator.FamilyGenerator {
 	)
 }
 
+func createPodStatusContainerReadyTimeFamilyGenerator() generator.FamilyGenerator {
+	return *generator.NewFamilyGeneratorWithStability(
+		"kube_pod_status_container_ready_time",
+		"Readiness achieved time in unix timestamp for a pod containers.",
+		metric.Gauge,
+		basemetrics.ALPHA,
+		"",
+		wrapPodFunc(func(p *v1.Pod) *metric.Family {
+			ms := []*metric.Metric{}
+
+			for _, c := range p.Status.Conditions {
+				if c.Type == v1.ContainersReady {
+					ms = append(ms, &metric.Metric{
+						LabelKeys:   []string{},
```

Comment:
Will this require Label Keys identifying the container?
Similar to `createPodContainerInfoFamilyGenerator`

Reply:
Yes. I'm also going to look at the names of the metrics and better align them with the conventions I'm seeing in other `kube_pod_*` metrics. 

Other than requested changes, this is a copy of the other PR. I will take this opportunity to adjust some things and better align it with the rest of the project.

Reply:
My understanding is it's not that a specific container is ready, but all the containers are ready. If we emitting a metric about the status of individual containers, I would add container id so it could be joined to `kube_pod_container_info`. Thoughts?

Reply:
Yes, you're right! I was thinking it's using something in .status.containerStatuses which would be per container. 

```
  - lastProbeTime: null
    lastTransitionTime: "2022-11-18T18:50:53Z"
    status: "True"
    type: Ready
  - lastProbeTime: null
    lastTransitionTime: "2022-11-18T18:50:53Z"
    status: "True"
    type: ContainersReady
```